setup.cfg

```diff
@@ -42,6 +42,10 @@ install_requires=
     psutil
     scipy
     scikit-learn
+    miplib
+    quoll
+    toupy
+    scikit-image
```

Comment:
are we happy leaving these on a 'latest version' or do we thing it might be good to lock them to the version working here and then review in the futuer?

Reply:
Thanks for the review, Nick! I'm going to remove toupy and scikit-image libraries from the extra dependencies